app/views/shared/_navbar.html.erb

```diff
@@ -1,39 +1,32 @@
+  <%# 4 icons + one collapse button in the navbar %>
   <div class="container-fluid">
-    <%= link_to "#", class: "navbar-brand" do %>
-      <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+    <a href="/"><i class="fas fa-home"></i></a>
+    <a href="/items"><i class="fas fa-search"></i></a>
+    <a href="/items/new"><i class="far fa-plus-square"></i></a>
+    <a href="/chatrooms"><i class="far fa-comment"></i></a>
 
+    <%# collapse button shows the avatar image (for the demo user only) or a normal icon (for everyone else) %>
     <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-      <span class="navbar-toggler-icon"></span>
+      <% if current_user == User.first %>
+        <%= image_tag "demo_user.png", class: "avatar dropdown-toggle text-white", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+      <% else %>
+        <a href="#" class="avatar dropdown-toggle text-white" id="navbarDropdown" data-bs_toggle="dropdown" aria-haspopup=true aria-expanded=false>
+          <i class="far fa-user"></i>
+        </a>
+      <% end %>
     </button>
 
-    <div class="collapse navbar-collapse" id="navbarSupportedContent">
+    <%# collapse menu to show 'Login' or 'Log out' option. %>
+    <div class="collapse navbar-collapse my-1" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto">
         <% if user_signed_in? %>
           <li class="nav-item active">
-            <%= link_to "Home", "#", class: "nav-link" %>
-          </li>
-          <li class="nav-item">
-            <%= link_to "Messages", "#", class: "nav-link" %>
-          </li>
-          <li class="nav-item dropdown">
-            <%# change the avatar image for the demo user only %>
-            <% if current_user == User.first %>
-              <%= image_tag "demo_user.png", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
-            <% else %>
-              <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { bs_toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
-            <% end %>
-            <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
-              <%= link_to "Action", "#", class: "dropdown-item" %>
-              <%= link_to "Another action", "#", class: "dropdown-item" %>
-              <%= link_to "Log out", destroy_user_session_path, method: :delete, class: "dropdown-item" %>
-            </div>
+            <%= link_to "Log out", destroy_user_session_path, method: :delete %>
```

Comment:
If we want to login / logout, it would make more sense to do it by clicking on the profile icon

Reply:
ok, removed the profile picture as requested

Reply:
So, if we click on the profile icon, there's a dropdown to logout ? But what if the user want to go to heir profile page, is there a route for that or we will create it later ?

Reply:
There can be two ways, we can:
1. Add "Profile Page" above "Log out" in the dropdown menu; or
2. Remove the dropdown menu and replace the url by user profile page

I'm fine with both options!

Reply:
Ok, let's talk about it tomorrow. I'll merge the branch :)